pkg/cluster/rook_ceph_test.go

```diff
@@ -3,11 +3,17 @@ package cluster
 import (
 	"context"
 	_ "embed"
+	"fmt"
+	"reflect"
 	"testing"
 
 	"github.com/blang/semver"
+	"github.com/golang/mock/gomock"
```

Comment:
I need to mock an interface for testing. Do you have something against this package or mocking in general? What would you like to see instead?

Reply:
In the projects I have been working with a policy to avoid us much as possible debs to try make things easier to keep maintained. I did not use gomock so far and I can not be opinionated in this way. I am ok with 👍 

In the future if we found a better option we can revisit this one.